docs/development.md

```diff
@@ -99,7 +103,21 @@ The dev version of the manifests can be generated with
 
 1. [Install the gcloud cli][gcloud_sdk].
 2. Set project: `gcloud config set project YOUR_PROJECT_NAME`.
-3. Build & push dev images: `make docker-push-dev`.
+3. To build images with custom tags, edit [.bazelrc](./.bazelrc) as follows:
```

Comment:
don't modify `.bazelrc`, use make with environment variables. If a build target is missing the ability to customize the Bazel command please file an issue

Reply:
I was able to pass the custom tag and the registry on the command line. Will document that approach.

Reply:
fixed